pkg/github/checker.go

```diff
@@ -34,7 +34,11 @@ func (e *CheckEvent) runChecks(filters ...filter.Option) {
 	}
 	defer git.RemoveRepo(dir)
 
-	var wg sync.WaitGroup
+	// SYNCHRONIZATION
+	// Wait group for waiting all tasks to be done in the end
+	var tasksDone sync.WaitGroup
+	// Chan allowing to run only n goroutines at the same time
+	currentlyRunning := make(chan int, e.subFolderParallelism)
```

Comment:
good first implementation. In the end you will probably need to do it application wide and not per checkEvents. because if somebody creates a mass PR each PR will trigger `e.subFolderParallelism` jobs and your bot will die

Reply:
Yes maybe, I guess it's good enough for now. We'll see with more experience